### DISCUSSION TITLE
ios自定义短语是否考虑后续支持导入

### DISCUSSION THREAD

**Discussion by Vaxjw** [imfuxiao/Hamster#460] [Q&A]
用户自定义的短语

**Comment by imfuxiao** [comment]
您好，是指导入 iOS 系统的自定义短语么？

**Reply by Vaxjw** [reply]
是的

**Reply by imfuxiao** [reply]
嗯嗯，这个功能可以做，但是会晚些在开发。我记得 issue 中有相关的需求。

**Comment by groverlynn** [comment]
如果有MacOS設備的話，從Mac導出（https://support.apple.com/zh-cn/guide/mac-help/mchl2a7bd795/mac），然後導入Rime中。
